.github/workflows/kind.yml

```diff
@@ -95,6 +95,7 @@ jobs:
       run: |
         make test-coverage
         make coverage-verify
+        make gosec-scan
```

Comment:
Maybe this should just have this step match `stolostron`? With the change that we `tee` to the output file rather than having `-out gosec.json` in the command? (`tee` because otherwise I don't think we'll see the output?)
```
    - name: Test Coverage and Report Generation
      if: ${{ matrix.kind == 'latest' }}
      run: |
        make test-coverage | tee report_unit.json
        make coverage-verify
        make gosec-scan | tee gosec.json
```

Reply:
We could add `-stdout` if we also want the report printed. I think the "normal" output of the tool is some logs, and not the report. So `tee` might make the report unreadable by other tools.

Reply:
Oh nice--I was wondering about other output. Then yes let's use `-stdout`. Otherwise, the scan might fail with no indication of why.

Reply:
Also, now that this is blocking, maybe we should move it up with the linting so that it can be addressed more quickly?